src/components/Navbar/style.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/prefer-default-export */
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles({
+  nav: {
+    color: '#737373',
+    fontWeight: '700',
+    textTransform: 'capitalize',
+  },
+  button: {
+    color: 'white',
+    background: '#23A6F0',
+    fontWeight: '700',
+    boxSizing: 'border-box',
+    padding: '13px 30px',
+  },
+  toolbar: {
+    justifyContent: 'space-evenly',
+  },
+  login: {
+    color: '#23A6F0',
+    fontWeight: '700',
+    textTransform: 'capitalize',
+    marginRight: '-170px',
```

Comment:
try to avoid negative margin as it may break further styles if project will grow. Find another solution for this